dashboard/app/models/levels/level.rb

```diff
@@ -79,6 +79,7 @@ class Level < ApplicationRecord
     hint_prompt_attempts_threshold
     short_instructions
     long_instructions
+    custom_instructions
```

Comment:
Should this be something more specific to AI Lab? Long and short instructions are custom instructions so this name seems confusing to me.

Reply:
Oh, that should have been `dynamic_instructions` (`custom_instructions` was a previous name I tried) but maybe this line isn't even needed?